packages/webcert/src/store/certificate/certificateReducer.ts

```diff
@@ -1,5 +1,4 @@
 import {
```

Comment:
Is the deletion here relevant? Isn't this to do with another feature?

Reply:
Someone had forgotten to remove it and if I didn't remove it, I got an error in the browser

Reply:
Ah, saw now that it's also in the master branch, and is never used there, so probably from some unmerged branch?
I'll approve the PR